### ISSUE TITLE
Deno Lint fails with thread 'tokio-runtime-worker' has overflowed its stack

### ISSUE THREAD

**Issue by khrome83** [denoland/deno#7279]
Since upgrading to Deno 1.3.2 I have seen an overflow error running `deno lint --unstable`

```
thread 'tokio-runtime-worker' has overflowed its stack
fatal runtime error: stack overflow
/home/runner/work/_temp/998ca1c5-1210-471d-bd26-b595fc5f0f07.sh: line 1:  4053 Aborted                 (core dumped) deno lint --unstable
##[error]Process completed with exit code 134.
```
You can see the error here - 
https://github.com/use-seedling/seedling/runs/1048772375?check_suite_focus=true

To reproduce, download master of https://github.com/use-seedling/seedling/
Run `deno upgrade --version 1.3.2`
Run `deno lint --unstable` 

You can see the issue. No error file is output, so I have nothing else to give to show the issue.
The repository did have a lot of warnings for `ban-types`. I have either fixed or added `//deno-lint-ignore ban-types`.

**Comment by bartlomieju** [comment]
Thanks for the report @khrome83 I verified that it's indeed the case even after #7276. Could you try to isolate which file causes the overflow? 

CC @kdy1

**Comment by khrome83** [comment]
I have `src/css/form.ts` which is very similar as well. I believe it's because of the `ban-types`. I used a `// deno-lint-ignore`.

**Comment by bartlomieju** [comment]
@khrome83 it's not problem with directive, I think somethings wrong in the parser.

**Comment by bartlomieju** [comment]
Fixed in #7402
